index.html

```diff
@@ -885,6 +831,103 @@ <h5>Timing Properties</h5>
               The end time of a <a>DAPT Script</a> is for practical purposes the end of the <a>Related Media Object</a>.
             </p>
           </section>
+
+          <section id="sharedprop-represents">
+            <h5>Represents</h5>
+            <p>The <dfn>Represents</dfn> property
+              indicates which components of the <a>related media object</a>
+              the contents of the parent object represent.
```

Comment:
"parent object" means "current object" right?

Reply:
This has changed now.